### DISCUSSION TITLE
Community profile is missing important files

### DISCUSSION THREAD

**Discussion by BobinMathew** [fastapi/fastapi#6853] [Questions]
![image](https://user-images.githubusercontent.com/32351527/108369044-25499a80-7221-11eb-82ad-dc94342d8a7a.png)
I see your community profile is missing this files
Can I send a PR

**Comment by tiangolo** [comment]
Not sure where you are getting that dashboard from, but for now I'll pass on this suggestion, thanks for the interest!

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
